add-chain/main.go

```diff
@@ -255,7 +255,19 @@ func getGasPayingToken(l1rpcURl string, SystemConfigAddress superchain.Address)
 		}
 		return nil, err
 	}
+	if !usingCustomGasToken {
+		return nil, nil
+	}
 
+	result, err := sc.GasPayingToken(&opts)
+	if err != nil {
+		if strings.Contains(err.Error(), "execution reverted") {
+			// This happens when the SystemConfig contract
+			// does not yet have the CGT functionality.
+			return nil, nil
+		}
+		return nil, err
+	}
```

Comment:
Do we need to check for execution reverted here? If the SystemConfig contract does not yet have the CGT functionality, wouldn't we detect that on L251?

Reply:
I'm not really sure. Would have to look through SystemConfig.sol source code. Probably a safe assumption but not sure if there is a case where one of those functions existed without the other. Safer to leave the additional error handling in now and can remove later if/when we confirm